### ISSUE TITLE
Question about Training SGFN

### ISSUE THREAD

**Issue by J-C-Y** [ShunChengWu/3DSSG#60]
Hi Thanks for your great work!

I have some questions about training SGFN using 3RSCAN dataset.

I run the train with the code that you provided.

` python main.py --mode train --config [PATH to config_SGFN_full_l160]
`

But the loss did not decrease and remained around 5.

I checked that loss_rel was reduced to around 0.0x, but the loss_obj remained around 5.

What's the problem? Can you check the code? 

I would appreciate it if you could share how the loss fell.

-  Overall Loss

![image](https://github.com/user-attachments/assets/61978b02-465d-4055-a2ea-7f2fbdb46c21)

 - Obj Loss

![image](https://github.com/user-attachments/assets/cf06fc4e-2206-481b-a62e-9b8ac86cf495)

- Rel Loss

![image](https://github.com/user-attachments/assets/5600b30c-9914-4963-9c0b-368a846ba06b)

- Confusion Matrix of Node <- It only predicts to one object
![image](https://github.com/user-attachments/assets/6f8ec19f-b3d1-4640-b97a-811c208be0cc)

**Comment by ShunChengWu** [comment]
Hm.. there are many possibilities. 
It's possible that the dataset is wrong or any other settings are incorrect. It's also possible it's the initialization problem of the model weight. Can you try to run the pre-trained model to see if your setup are correct?
